harfbuzz-sys/Cargo.toml

```diff
@@ -46,5 +46,6 @@ optional = true
 
 [features]
 default = ["build-native-harfbuzz", "build-native-freetype"]
-build-native-harfbuzz = ["cc", "pkg-config"]
+build-native-harfbuzz = ["cc"]
 build-native-freetype = ["freetype", "freetype/freetype-sys"]
+use-pkg-config-harfbuzz = ["pkg-config"]
```

Comment:
There are a couple issues here:

1. This changes  the default behavior of the crate. Before it would try to use the system version of HarfBuzz unless you set an environment variable. I don't think we can do this without bumping the major version number.  I think the default behavior should follow what most `-sys`-type crates do, whatever that is.
2. This feature shouldn't refer to `pkg-config`, because `pkg-config` is an implementation detail of how you find the system shared library. The most important thing here is that a feature like this controls whether or not we are dynamically linking to the system library or including a statically compiled version of HarfBuzz. That's why I chose the `static` name when I tried to make this configurable as a feature. We could discuss a better name, but maybe this change could be written on top of mine?